ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/mpls.j2

```diff
@@ -8,6 +8,8 @@
 {% for loopback_interface in loopback_interfaces | arista.avd.natural_sort %}
 {%     if loopback_interfaces[loopback_interface].mpls is arista.avd.defined %}
 {%         set mpls_configured.loopback_interfaces = true %}
+{%     elif loopback_interfaces[loopback_interface].node_segment is arista.avd.defined %}
```

Comment:
This logic is correct but we can add ```elif``` condition to the same ```if``` condition by using an ```or``` in between two arista.avd.defined tests. That makes the code more clear because we are setting ```true``` in both cases.

Reply:
I was referring to similar coding style as in documentation.

Reply:
Fixed.